doc/tutorial/climada_engine_unsequa.ipynb

```diff
@@ -129,7 +129,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    " **Several helper methods exist to make generic InputVar for Exposures, ImpactFuncSet, Hazard, Entity (including DiscRates and Measures)**. These are described in details in the tutorial [climada_engine_uncertainty_helper](climada_engine_uncertainty_helper.ipynb). These are a good bases for your own computations."
+    " **Several helper methods exist to make generic InputVar for Exposures, ImpactFuncSet, Hazard, Entity (including DiscRates and Measures)**. These are described in details in the tutorial [climada_engine_uncertainty_helper](https://climada-python.readthedocs.io/en/latest/tutorial/climada_engine_unsequa_helper.html). These are a good bases for your own computations."
```

Comment:
```suggestion
    " **Several helper methods exist to make generic InputVar for Exposures, ImpactFuncSet, Hazard, Entity (including DiscRates and Measures)**. These are described in details in the tutorial [Helper methods for InputVar](climada_engine_unsequa_helper.ipynb). These are a good bases for your own computations."
```
The relative path to the notebook was not a problem, but the wrong file name was.